internal/controllers/nmc_reconciler_test.go

```diff
@@ -143,6 +158,9 @@ var _ = Describe("NodeModulesConfigReconciler_Reconcile", func() {
 			reflect.TypeOf((*context.Context)(nil)).Elem(),
 		)
 
+		kubeClient.EXPECT().Get(ctx, types.NamespacedName{Name: nmc.Name}, &node).Return(err)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
What are we testing here? If `err` was not initialized, then it will be `nil` by default.

Reply:
fixed, moved `kubeClient.EXPECT().Get(ctx, types.NamespacedName{Name: nmc.Name}, &node).Return(err)` to the InOrder section (`Expect(err).NotTo(HaveOccurred())` should not be there indeed)